src/Utils/OccRunner.php

```diff
@@ -55,7 +55,7 @@ public function run($command, $args = [], $asJson = false){
 			$cmdLine = trim($command . ' ' . $extra);
 			foreach ($args as $optionTitle => $optionValue){
 				if (strpos($optionTitle, '--') === 0){
-					$line = trim("$optionTitle $optionValue");
+					$line = trim("$optionTitle=$optionValue");
```

Comment:
Hmm, I was mostly thinking that the key and value need to be escaped separately. But if you think that the current way is fine, then okay.